README.md

```diff
@@ -68,21 +72,30 @@ app.post('/', async (req, res) => {
 })
 ```
 
-
 ## Plugin options
 
 `disposeOnClose` - automatically invoke configured `dispose` for app-level `diContainer` hooks when the fastify instance is closed.
-  Disposal is triggered within `onClose` fastify hook.
-  Default value is `true`
+Disposal is triggered within `onClose` fastify hook.
+Default value is `true`
 
 `disposeOnResponse` - automatically invoke configured `dispose` for request-level `diScope` hooks after the reply is sent.
-  Disposal is triggered within `onResponse` fastify hook.
-  Default value is `true`
+Disposal is triggered within `onResponse` fastify hook.
+Default value is `true`
+
+`asyncInit` - whether to process `asyncInit` fields in DI resolver configuration. Disabling this will make app startup slightly faster.
+Default value is `false`
```

Comment:
I assume this means the new release is a major?

Reply:
I don't think that is necessary, by default everything is disabled and works same way as it did before.